src/main/java/fr/techad/edc/popover/internal/swing/components/Popover.java

```diff
@@ -98,6 +100,38 @@ public void windowLostFocus(WindowEvent e) {
                 }
             }
         });
+
+        this.addMouseListener(new MouseListener() {
+            @Override
+            public void mouseClicked(MouseEvent e) {
+
```

Comment:
remove empty lines or use MouseAdapter to override the specific method (mouseExisted) and avoid to declare empty methods